text/0000-nft-standard.md

```diff
@@ -0,0 +1,151 @@
+
+If the NFT requires a metadata, the output script data saved on blockchain **MUST** be the metadata URL in an immutable protocol (e.g. ipfs as explained in the last section), e.g. ipfs://ipfs/<hash>/metadata.json.
+
+The metadata should have the following JSON structure:
+
+```
+{
+    "name": {
+        "type": "string",
+        "description": "Identifies the asset to which this token represents"
+    },
+    "description": {
+        "type": "string",
+        "description": "Describes the asset to which this token represents"
+    },
+    "file": {
+        "type": "string",
+        "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
+    },
+    "attributes": {
+        "type": "Array<AttributeObject>",
+        "description": "These are the extra attributes for the digital asset. It's an array of AttributeObject to make it as flexible as possible."
+    },
+}
+```
+
+The AttributeObject has the following JSON structure:
+
+```
+{
+    "type": {
+        "type": "string",
+        "description": "Type of the attribute, e.g. rarity, stamina, eyes"
+    },
+    "value": {
+        "type": "string | decimal",
+        "description": "Value of the attribute, e.g. rare, 1.4, blue"
+    }
+}
+```
+
+The AttributeObject may have more attributes than the ones described above but those are the required ones.
+
+### Metadata example
+
+```
+{
+    "name": "Gandalf",
+    "description": "A wizard, one of the Istari order, and the leader and mentor of the Fellowship of the Ring",
+    "file": "ipfs://ipfs/QmbuthvFV2EjvfmWXxt2L83PwPPwbjjggBhVsrEB7AXW123/gandalf.png",
+    "attributes": [
+        {
+            "type": "rarity",
+            "value": "super rare"
+        },
+        {
+            "type": "hp",
+            "value": 32
+        },
+        {
+            "type": "intelligence",
+            "value": 99
+        },
+        {
+            "type": "ring",
+            "value": 0
+        }
+    ]
+}
+```
+
+## Deposit & Fee
+
+### Deposit
+
+The deposit will be the same as in the normal token creation, i.e. 1% of the amount created deposited in HTR.
+
+For example, if you'd like to create 100 units of a NFT, you must deposit 0.01 HTR. 1% of 100 units is 1 unit and given that we never handle decimals, only integers, 1 unit of HTR is 0.01 HTR.
+
+- 500 NFT units: deposit of 0.05 HTR.
+- 100 NFT units: deposit of 0.01 HTR.
+- 10 NFT units: deposit of 0.01 HTR.
+- 1 NFT unit: deposit of 0.01 HTR.
+
+This deposit may be returned in case of a melt of the created NFTs. Important to notice that you have the deposit back only if you melt an amount bigger than or equal to 100 units, just like the normal custom token.
+
+### Fee
+
+The first output of a NFT transaction will contain the script with the data string. This output must have some value and we use 0.01 HTR for that. It's an output that can never be returned.
+
+#### Why use HTR for the fee and not the custom token created?
```

Comment:
I feel it should be moved to an FAQ section. It seems move like a curiosity than part of the standard.